prime-router/docs/design/proposals/0023-condition-to-code-mapping/0023-condition-to-code-mapping.md

```diff
@@ -0,0 +1,229 @@
+
+1.) Utility to check sender test compendiums against current mapping table<br>
+2.) Utility to automatically update tables from Value Set Authority Center API (https://www.nlm.nih.gov/vsac/support/usingvsac/vsacfhirapi.html) <br>
+3.) Reporting mechanism to inform RS/sender when unmapped value was found.<br>
+4.) Mechanism to automate which conditions each state considers reportable.
+
+## Design
+
+
+### Condition Mapping Table
+
+The condition mapping table will be made up of CSTE ValueSets and contain the following columns:
+
+| Column Name                   | Description                                   | Example                                                                        |
+|-------------------------------|-----------------------------------------------|--------------------------------------------------------------------------------|
+| Member OID                    | ValueSet Identifier                           | 2.16.840.1.113762.1.4.1146.239                                                 |
+| Name                          | ValueSet Name                                 | Chlamydia trachomatis Infection (Tests for Chlamydia trachomatis Nucleic Acid) |
+| Code                          | LOINC or SNOMED coded value                   | 16601-7                                                                        |
+| Descriptor                    | LOINC or SNOMED term descriptio               | Chlamydia trachomatis rRNA [Presence] in Urine by Probe                        |
+| Code System                   | Indicates whether code is LOINC or SNOMED     | LOINC                                                                          |
+| Version                       | LOINC or SNOMED release version               | 2.74                                                                           |
+| Status                        | Indicates if code is active or deprecated     | Active                                                                         |
+| Condition Name                | Name of associated reportable condition       | Chlamydia trachomatis infection (disorder)                                     |
+| Condition Code                | SNOMED value associated with condition        | 240589008                                                                      | 
+| Condition Code System         | System used for condition code                | SNOMEDCT                                                                       |
+| Condition Code System Version | SNOMED version associated with condition code | 2023-03                                                                        |
+
+The column names are taken directly from the RCTC spreadsheet.Both LOINC and SNOMED codes are combined in this table and can be identified by column "Code System".
```

Comment:
```suggestion
The column names are taken directly from the RCTC spreadsheet. Both LOINC and SNOMED codes are combined in this table and can be identified by column "Code System".
```

Reply:
Can we link the spreadsheet here?

Reply:
I can, but it's behind a login. I'll see if I can add it to the google drive.